INSTALLATION.md

```diff
@@ -45,7 +45,15 @@ However, if you prefer a direct local installation, instructions follow.
 * Install the requirements.
 
   ```
-  python -m pip install -r binder/requirements.txt
+  python -m pip install -r binder/requirements-dev.txt
+  ```
+  ```
+  python -m pip install -r binder/requirements-preview.txt
+  ```
+  ```
+  python -m pip install -r binder/runtime.txt
```

Comment:
This file contains the line:

```
python-3.10
```

not a list of Python packages for pip.

Reply:
okee removing that one thanks